go/common/node/node.go

```diff
@@ -460,6 +460,11 @@ func (c *CapabilityTEE) Verify(ts time.Time) error {
 			return err
 		}
 
+		// Verify the quote.
+		if err = q.Verify(); err != nil {
+			return err
+		}
```

Comment:
why didn't we do this before? will adding this break things?

Reply:
It seems to work fine.

Reply:
This is actually not needed as `avrBundle.Open` already calls `validate` which calls `Verify` on the quote. So this was already being verified in `Open` and can be removed.